integration_test/go_java_launcher_integration_test.go

```diff
@@ -114,8 +143,21 @@ func TestSubProcessesParsedMonitorSignals(t *testing.T) {
 	assert.Len(t, trapped.FindAll(output.Bytes(), -1), 2, "expect two messages that SIGPOLL was caught")
 }
 
-func runMainWithArgs(t *testing.T, staticConfigFile, customConfigFile string) (string, error) {
-	output, err := mainWithArgs(t, staticConfigFile, customConfigFile).CombinedOutput()
+func runMainWithArgs(t *testing.T, staticConfigFile, customConfigFile string, env ...string) (string, error) {
+	jdkDir := "jdk"
+	javaHome, err := filepath.Abs(jdkDir)
+	if err != nil {
+		log.Fatalf("Failed to calculate absolute path of '%s': %v\n", jdkDir, err)
+	}
```

Comment:
nit: Given you have access to the `*testing.T` you can alternatively use `require.NoErrorf(t, err, "Failed to calculate ...", jdkDir)` instead of fatal logging.

Reply:
Done